cc/algorithms/count.h

```diff
@@ -115,6 +115,7 @@ class Count : public Algorithm<T> {
 
   uint64_t count_;
   std::unique_ptr<NumericalMechanism> mechanism_;
+  double delta_;
```

Comment:
I'm wondering if we should add `delta_` to `Algorithm`.  We can also add a second constructor where we can set both `epsilon_`, and `delta_`.  WDYT?

Reply:
Fixed in the next commit.